tools/astarte_e2e/lib/astarte_e2e/device_trigger.ex

```diff
@@ -0,0 +1,88 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule AstarteE2E.DeviceTrigger do
+  use Task, restart: :transient
```

Comment:
I think the default (temporary) is fine, we just want to perform one check and don't want to restart the process if it crashes (we want the whole application to go down instead)

```suggestion
  use Task
```